cmd/server/handler.go

```diff
@@ -208,6 +213,10 @@ func serve(c *config.Config, cmd *cobra.Command, handler http.Handler, address s
 		},
 	})
 
+	if tracer, _ := c.GetTracer(); tracer.IsLoaded() {
```

Comment:
Is it possible that there is an error here? If yes, tracer MAY be nil which will cause a panic

Reply:
At this point, a tracer would have been configured so it would not be possible for it to have an error - recall how we now exit if an error has occurred on boot up.

Although, for safety, incase future refactoring happens by other contributors, it's probably best to do error checking. Will add it in 👍